### PR TITLE
Warning in case of allergies

### PR DESCRIPTION
**Pull request by CecileSerene**
When displaying the product info, if the product contains any allergen the user is allergic to, it will display a warning pop-up message.

### CODE REVIEW

File: Components/Product.js [hunk 1]
```diff
@@ -128,11 +129,30 @@ class ProductScreen extends Component {
...
     }
 
+    _checkAllergies() {
+        const { product, isLoading} = this.state;
+        if (!isLoading) {
+            let user = UserService.findAll()[0];
+            let allergens = [];
+            for (let allergen of product.allergens_ids) {
+                if (Array.from(user.allergies).includes(allergen)) {
+                    allergens.push(allergen); //change to allergen name instead of id
```
Comment (CecileSerene) [inline Components/Product.js]: @dlphn Could you store in database the name of the allergens?

File: Components/Product.js [hunk 2]
```diff
@@ -128,11 +129,30 @@ class ProductScreen extends Component {
...
+                }
+            }
+            Alert.alert(
+                'Attention',
+                'Nous avons détecté des ingrédients dont vous êtes allergiques dans ce produit: '+ allergens.toString()
```
Comment (dlphn) [inline Components/Product.js]: allergique*

File: Components/Product.js [hunk 3]
```diff
@@ -128,11 +129,30 @@ class ProductScreen extends Component {
...
+        const { product, isLoading} = this.state;
+        if (!isLoading) {
+            let user = UserService.findAll()[0];
+            let allergens = [];
+            for (let allergen of product.allergens_ids) {
+                if (Array.from(user.allergies).includes(allergen)) {
+                    allergens.push(allergen); //change to allergen name instead of id
+                }
+            }
+            Alert.alert(
```
Comment (dlphn) [inline Components/Product.js]: Alert should only be triggered when allergens is not empty

img_1268

capture d ecran 2019-01-14 a 15 14 34

img_1269

Comment (CecileSerene) [reply Components/Product.js]: done

**Pull request merged**